app/routes.py

```diff
@@ -0,0 +1,350 @@
+from flask import Blueprint, jsonify, request, make_response
+from app.models.video import Video
+from app.models.customer import Customer
+from app.models.rental import Rental
+from app import db
+from datetime import datetime, timedelta
+
+from app.utilities import validate_request_body
+
+# assign videos_bp to the new Blueprint instance
+videos_bp = Blueprint("videos", __name__, url_prefix="/videos")
+# beginning CRUD routes/ endpoints for videos
+@videos_bp.route("", methods=["POST"])
+def post_one_video():
+    # request_body will be the user's input, converted to json. it will be a new record 
+    # for the db, with all fields (a dict)
+    request_body = request.get_json()
+    # this guard clause will give an error if user tries to submit request body that does
+    # not have all fields present
+    if 'title' not in request_body:
+        return make_response({"details": "Request body must include title."}, 400)
+    elif 'release_date' not in request_body:
+        return make_response({"details": "Request body must include release_date."}, 400)
+    elif 'total_inventory' not in request_body:
+        return make_response({"details": "Request body must include total_inventory."}, 400)
```

Comment:
you could create a function to handle these guard clauses dynamically and then use that function for customers and videos it could look something like:
```
def validate_parameters(request_body, parameters):
           for attribute in parameters:
                if attribute not in request_body:
                       jsonify({"details": f"Request body must include {attribute}."}, 400))
```